doc/js_tutorials/js_gui/js_image_display/js_image_display.markdown

```diff
@@ -60,15 +60,15 @@ ctx.putImageData(imgData, 0, 0);
 In addition, OpenCV-JavaScript implements image read and show using the above method. You can use cv.imread and 
 cv.imshow to read image from html canvas and display it.
 @code{.js}
-var img = cv.imread("canvas1");
-cv.imshow("canvas2", img);
+var img = cv.imread("inputImage");
```

Comment:
I would suggest to use `canvasInput` and `canvasOutput`, as they are actually canvas element.